src/unstable/components/Form/index.tsx

```diff
@@ -10,7 +9,9 @@ import { Box } from '../../../components/Grid';
 import FieldTemplate from './FieldTemplate';
 import BaseInput from './widgets/BaseInput';
 
-const baseWidgets = {
+const widgets: {
+	[k: string]: any;
```

Comment:
If we could have `FormWidgetProps` extend `react-jsonschema-form`'s `WidgetProps`, then this & `registerWidget`s `value` could be typed as `React.ComponentType<FormWidgetProps>`.

Reply:
When it comes to the Wdigets the types for `react-jsonschema-form`are a complete mess. Their own code doesn't work if you were to type it. I tried to get this to work for a few hours, but the juice is just not worth the squeeze. I'll probably revisit this in the future when the `Form`component moves out of beta.